vignettes/ospsuite.Rmd

```diff
@@ -11,96 +11,339 @@ vignette: >
+Call `print(object)` to display detailed information about any object.
+
+### Key Classes
+
+Understanding these main classes improves workflow effectiveness:
+
+**Simulation Objects:**
+
+- **`Simulation`**: Your loaded PBPK model from the .pkml file
+- **`SimulationResults`**: Output from running simulations
+- **`SimulationRunOptions`**: Controls for how simulations run (cores, error checking, progress bars)
+
+**Model Structure:**
+
+- **`Entity`**: Any part of the model (parameters, molecules, containers)
+- **`Container`**: Model compartments that contain other entities (organs, compartments)
+- **`Quantity`**: Entities with values (Parameters and Molecules)
+- **`Parameter`**: Model parameters you can modify
+- **`Molecule`**: Drug molecules and their concentrations
+
+**Results and Analysis:**
+
+- **`DataCombined`**: Container for combining simulation results and observed data
+- **`SimulationPKAnalyses`**: PK parameter calculations (AUC, Cmax, etc.)
+- **`Population`**: Virtual population data for population simulations
+
+**Configuration:**
+
+- **`OutputSchema`**: Defines when simulation results are saved
+- **`OutputSelections`**: Which quantities to include in results
+- **`SolverSettings`**: Numerical solver configuration
+
+## General Workflow Summary
+
+The typical ospsuite workflow follows these steps:
+
+1. **Load simulation** from .pkml file
+2. **Explore entities** (parameters, molecules, containers)
+3. **Modify values** (parameter values, initial concentrations)
+4. **Configure outputs** (what to track, when to save results)
+5. **Run simulation** 
+6. **Analyze results** (extract data, calculate PK parameters)
+7. **Visualize** (built-in plots or custom graphics)
+
+## Next Steps
+
+After mastering the basics, explore these advanced topics:
+
+### Essential Next Steps:
+- **[Loading a simulation and accessing entities](load-get.html)** - Deep dive into model exploration
+- **[Changing parameter and molecule start values](set-values.html)** - Advanced parameter manipulation
+- **[Running a simulation](run-simulation.html)** - Simulation options and batch processing
+
+### Population Studies:
+- **[Creating individuals](create-individual.html)** - Virtual patient creation
+- **[Population simulations](create-run-population.html)** - Large-scale population PK/PD
+
+### Analysis and Optimization:
+- **[PK Analysis](pk-analysis.html)** - Calculate pharmacokinetic parameters
+- **[Sensitivity analysis](sensitivity-analysis.html)** - Parameter importance analysis
+
+### Data Integration:
+- **[Working with observed data](observed-data.html)** - Import experimental data
+- **[DataCombined workflows](data-combined.html)** - Combine simulations with observations
+
+### Advanced Topics:
+- **[Efficient calculations](efficient-calculations.html)** - Performance optimization
+- **[Dimensions and Units](unit-conversion.html)** - Unit conversion and validation
+
+## General Information
```

Comment:
2 vignettes are not linked in this overview:
* Visualization with DataCombined
* Table Parameters